app/src/main/java/org/mozilla/tv/firefox/utils/ViewUtils.java

```diff
@@ -114,7 +116,13 @@ public static void showCenteredBottomToast(Context context, int resId) {
     }
 
     private static void showToast(Context context, int resId, String toastLocation) {
-        Toast toast = Toast.makeText(context, resId, Toast.LENGTH_SHORT);
+        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
```

Comment:
Inflating layouts are generally costly. Can we define a `WeakReference` to persist the toast layout (like a ViewHolder pattern)?

Reply:
After trying this, the toast visibly translate horizontally when a new message is shown, instead of always being centred. This doesn't look the greatest when a new toast is shown before the previous one has disappeared. Going to keep the current implementation for this reason.